### ISSUE TITLE
Does not appear to work in Blender 2.9x

### ISSUE THREAD

**Issue by orcinus** [vr-voyage/blender_bones_merger#5]
Getting the following when trying to enable the add-on in preferences:

![image](https://github.com/vr-voyage/blender_bones_merger/assets/1574258/f10a512c-1129-461c-8080-eb9c31944c69)

**Comment by vr-voyage** [comment]
Indeed, this is broken due to older Python not handling list components typing.

I tested a fix that I pushed here : 
https://github.com/vr-voyage/blender_bones_merger/blob/issue/1/fix292/bones_merger.py

Could you try to :
* Remove the addon
* Download that single python file
* Install it the same way (Add ons -> Install -> Choose the .py file you downloaded)
* Enable the installed add-on

And also, give it a test, see if everything seems OK.

**Comment by MiguVT** [comment]
@vr-voyage If possible close this issue, closing this will prevent have opened an issue that contributors can look searching to fix a thing that is already fixed.
Ty <3
